### PR TITLE
[RDAPI-19915] Update minimum system requirements in regards to cassandra prod use

### PR DESCRIPTION
**Pull request by dsantosaxway**
Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Update minimum system requirements in regards to cassandra prod use

## Deploy preview link

https://deploy-preview-2220--axway-open-docs.netlify.app/docs/apim_installation/apigtw_install/cassandra_install/#cassandra-prerequisites

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [x] You have verified that all status checks have passed

_Put an x in the boxes that apply. This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

File: content/en/docs/apim_installation/apigtw_install/cassandra_install.md [hunk 1]
```diff
@@ -43,6 +43,8 @@ API Gateway supports the following in a production environment:
 
 For details on requirements for high availability, see [Configure a Cassandra HA cluster](/docs/cass_admin/cassandra_config/).
 
+NOTE: General API Gateway prerequisite is a minimum configuration. A environment built on AWS s3.large instances may not be enough depending on the volume in production, in that case upgrade to m4.xlarge can be better to meet the high demand in the production environment.
```
Comment (kmercierwhite) [inline content/en/docs/apim_installation/apigtw_install/cassandra_install.md:46]: Notes should be in a {{< alert title="Note" >}} tag

I also edited some of the grammar here:

{{< alert title="Note" >}}The prerequisites are a minimum configuration. A environment built on AWS s3.large instances may not be enough depending on the volume of data in production. In this case, an upgrade to m4.xlarge can be better to meet the high demand in a production environment.{{< /alert >}}

File: content/en/docs/apim_installation/apigtw_install/cassandra_install.md [hunk 2]
```diff
@@ -51,7 +53,27 @@ The default API Gateway installation includes a 64-bit OpenJDK JRE (`apigateway/
 
 Cassandra is designed to run on commodity distributed drives, and therefore it is strongly recommended not to use a storage area network (SAN) for Cassandra deployments.
 
-For more information on Cassandra hardware choice recommendations, see [Hardware choices]( https://cassandra.apache.org/doc/latest/operating/hardware.html).
+For more information on Cassandra hardware choice recommendations, see [Hardware choices](https://cassandra.apache.org/doc/3.11/cassandra/operating/hardware.html).
+
+AWS General Recommendations
```
Comment (kmercierwhite) [inline content/en/docs/apim_installation/apigtw_install/cassandra_install.md:58]: This should probably be a title

**Review by kmercierwhite — CHANGES_REQUESTED** [review]

File: content/en/docs/apim_installation/apigtw_install/cassandra_install.md (hunk 1 above)
Comment (dsantosaxway) [reply content/en/docs/apim_installation/apigtw_install/cassandra_install.md]: Done!

File: content/en/docs/apim_installation/apigtw_install/cassandra_install.md (hunk 2 above)
Comment (dsantosaxway) [reply content/en/docs/apim_installation/apigtw_install/cassandra_install.md]: Done!

**Review by kmercierwhite — APPROVED** [review]
Thanks Diogo!

**Pull request merged**